uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/files.xml

```diff
@@ -10,4 +10,5 @@
     <xi:include href="PublicHoliday.xml" parse="xml" />
```

Comment:
You mean the PublicHoliday.xml or the whole files.xml? If mean the whole file, it's kind of index file listing all the schemas for this extension. This is then added to the CiviCRM own index and it's what GenCode.php reads to generate the DAOs